services/QuillLMS/app/assets/stylesheets/pages/privacy_and_security.scss

```diff
@@ -0,0 +1,261 @@
+.privacy-and-security-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  .divider-line {
+    width: 200px;
+    height: 1px;
+    background-color: $quill-grey-25;
+  }
+  .header-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 96px;
+    h1 {
+      color: $quill-green;
+      text-align: center;
+      font-family: "Adelle Sans";
```

Comment:
I think we want to avoid including font-family here if it's Adelle Sans -- there's some difference between "adelle-sans" and "Adelle Sans" that occasionally results in weird rendering.

Reply:
yeah good call, I'm noticing that on mobile now